api/api.go

```diff
@@ -397,16 +397,158 @@ func (serv *MetadataServer) GetModels(stream pb.Api_GetModelsServer) error {
 	}
 }
 
+func (serv *MetadataServer) ListUsers(in *pb.Empty, stream pb.Api_ListUsersServer) error {
+	proxyStream, err := serv.meta.ListUsers(stream.Context(), in)
+	if err != nil {
+		return err
+	}
+	for {
```

Comment:
I believe sam commented on this, but you can put this whole for-loop into a function an reuse it for most of these

Reply:
i tried doing that before, but couldn't since the streams are all considered different types (api user stream is a different type than api features stream). i tried making one with a generic type, but it got weirdly complicated